source/gameengine/Ketsji/KX_SG_NodeRelationships.cpp

```diff
@@ -62,15 +65,20 @@ bool KX_NormalParentRelation::UpdateChildCoordinates(SG_Node *child,
     return true;  // false;
   }
   else {
-    // the childs world locations which we will update.
-    const MT_Vector3 &p_world_scale = parent->GetWorldScaling();
-    const MT_Vector3 &p_world_pos = parent->GetWorldPosition();
-    const MT_Matrix3x3 &p_world_rotation = parent->GetWorldOrientation();
-
-    child->SetWorldScale(p_world_scale * child->GetLocalScale());
-    child->SetWorldOrientation(p_world_rotation * child->GetLocalOrientation());
-    child->SetWorldPosition(p_world_pos +
-                            p_world_scale * (p_world_rotation * child->GetLocalPosition()));
```

Comment:
Isn't the p_world_scale multiplication here the issue? If I am not wrong removing that multiplication would make this more or less equivalent of the pull request, because the GetXXXTransform() returns a scaled transform, while the GetXXXPosition and GetXXXPosition return un-scaled data.

Reply:
I did tests removing the p_world_scale but it didn't fix totally the issue.